app/scripts/sagas/index.js

```diff
@@ -128,6 +132,38 @@ export function * fetchPeerIds () {
   }
 }
 
+export function * loadConfig (action) {
+  try {
+    const ipfsConfig = yield call(api.getConfig)
+    const configJSON = JSON.stringify(ipfsConfig, undefined, 2)
```

Comment:
I currently have two properties that store config state: `config` and `draft`. `config` is the original and draft contains user changes. If the user clicks reset I set `draft` = `config`.

Would it be acceptable here to store `draft` as a string (since it's unvalidated user input), and `config` as object? I'm not sure how else I could handle this, so I'm open to suggestions.